packages/frontend/components/ui/checkbox.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Checkbox({
+  name,
+  checked,
+  type,
+  disabled,
+  className,
+  value,
+  label,
+  required,
+  onChange,
+  ...otherProps
+}) {
+  function changeValue(targetValue) {
+    if (onChange) {
+      onChange(targetValue);
+    }
+  }
+
+  let customStyleName = '';
+  if (checked && type === 'success') {
+    customStyleName = 'success-check-box';
+  } else if (checked && type === 'danger') {
+    customStyleName = 'danger-check-box';
+  }
+
+  return (
+    <div className="flex items-center">
+      <input
+        id={name}
+        name={name}
+        type="checkbox"
+        checked={checked}
+        className={cls(`default-check-box ${customStyleName}`, className)}
+        disabled={disabled}
+        onChange={e => changeValue(e.currentTarget.checked)}
+        required={required}
+        {...otherProps}
+      />
+      <label
+        htmlFor={name}
+        className="text-label-color text-base font-normal checkbox-label"
+      >
+        {label}
+        {required && <span className="ml-1">*</span>}
+      </label>
+    </div>
+  );
+}
+
+Checkbox.defaultProps = {
+  className: '',
+  checked: false,
+  disabled: false,
+  required: false,
+  type: '',
+  label: '',
+};
+
+Checkbox.propTypes = {
+  className: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  disabled: PropTypes.bool,
+  checked: PropTypes.bool,
+  label: PropTypes.string,
+  required: PropTypes.bool,
+  type: PropTypes.string,
```

Comment:
PropTypes are really important to understanding which prop is doing what. Giving `PropTypes.string` for `type` isn't explain itself, you can check [how we write it in buttons.](https://github.com/mustaphaturhan/ucurtma-app/blob/0c64c73c0296fa891f91671c129b47bba5671663/packages/frontend/components/ui/button.js#L64)

my suggestion is: `type: PropTypes.oneOf(['success', 'danger'])`